test/utils.spec.js

```diff
@@ -0,0 +1,38 @@
+import {flatten, throwInvariant, isPromiseLike} from '../src/utils';
```

Comment:
do we want to add tests for `mapValues` and `promisify`?

Reply:
those have been removed in this branch (really in this 4.0 branch, which this branched off of)